admin/admin.go

```diff
@@ -0,0 +1,181 @@
+package admin
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/google/uuid"
+	"github.com/vocdoni/blind-csp/model"
+	"go.vocdoni.io/dvote/httprouter"
+	"go.vocdoni.io/dvote/httprouter/bearerstdapi"
+	"go.vocdoni.io/dvote/log"
+)
+
+var BearerTokenRequestsAmount int64 = 100000000
+
+type Admin struct {
+	router             *httprouter.HTTProuter
+	api                *bearerstdapi.BearerStandardAPI
+	storage            *model.MongoStorage
+	electionController *ElectionController
+	userController     *UserController
+}
+
+func NewAdmin() (*Admin, error) {
+	admin := new(Admin)
+
+	var err error
+
+	// Storage
+	admin.storage = &model.MongoStorage{}
+	if err := admin.storage.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	// Models & Controllers
+	electionStore := model.NewElectionStore(admin.storage)
+	admin.electionController = NewElectionController(electionStore)
+
+	userStore := model.NewUserStore(admin.storage)
+	admin.userController = NewUserController(userStore)
+
+	return admin, err
+}
+
+func (admin *Admin) ServeAPI(r *httprouter.HTTProuter, baseRoute string) error {
+	if len(baseRoute) == 0 || baseRoute[0] != '/' {
```

Comment:
```suggestion
	if !strings.StartsWith(baseRoute, "/") {
```

Reply:
Actually changed for if !strings.HasPrefix(baseRoute, "/") as StartsWith errored as unidefined